### DISCUSSION TITLE
如何将MKdocs部署到Vercel？

### DISCUSSION THREAD

**Discussion by Wcowin** [w1ndys/easy-qfnu-wiki-v2#111] [Q&A] [answered]
我尝试了但是失败了

**Comment by w1ndys** [comment]
在Vercel配置面板

~构建命令`pip install -r requirements.txt && mkdocs build`~

> 这边昨天写错了（也不算错），Vercel有更细分的配置，他把下载命令单独开了一个配置项
> installcommand：pip install -r requirements.txt
> buildcommand：mkdocs build

输出目录`site`

还可以设置环境变量等

还有一种方法就是在GitHub输出gh-pages分支，然后用Vercel部署gh-pages分支，纯静态文件不需要构建命令

**Comment by w1ndys** [comment, marked as answer]
你也可以通过`Vercel.json`来实现，这样可以在切换Vercel的时候，免去配置面板操作，更加方便灵活
具体可以看[Vercel.json官方文档](https://vercel.com/docs/projects/project-configuration#buildcommand)

**Reply by w1ndys** [reply]
只是用来部署的话应该只配置下面三个命令即可
https://vercel.com/docs/projects/project-configuration#buildcommand
https://vercel.com/docs/projects/project-configuration#installcommand
https://vercel.com/docs/projects/project-configuration#outputdirectory
分别对应构建命令，下载命令，输出目录

**Comment by Wcowin** [comment]
<img width="1331" alt="image" src="https://github.com/W1ndys/Easy-QFNU/assets/99159173/b8b7b094-c1ea-4b09-853a-333d5e95792c">
哥们你好，依旧是我，我仍然对Vercel的部署感到疑惑，我无法找到输出的gh-pages分支
<img width="837" alt="image" src="https://github.com/W1ndys/Easy-QFNU/assets/99159173/60f841a8-d928-48b2-9bb5-08115aa70015">

**Reply by w1ndys** [reply]
settings>Git﹥Production Branch

把对应分支填进去保存

然后重新deploy一下即可
